src/main/webapp/assets/feedScript.js

```diff
@@ -77,15 +77,24 @@ document.addEventListener('DOMContentLoaded', onLoad);
 /**
  * Fires as soon as the DOM is loaded.
  */
-function onLoad() {
+async function onLoad() {
+  const collegeid = (new URLSearchParams(window.location.search)).get('collegeid');
+
+  // Don't load the map, posts if there wasn't a college ID provided.
+  if (!collegeid) {
+    document.getElementById('find-events-title').innerHTML =
+      'No college provided. Please visit the <a href="/">home page</a> to pick a college.';
```

Comment:
Another possibility -- if there is no college provided we could just redirect them back to the main page?

Reply:
I guess my vote goes to option (D)

Reply:
Also another option - keep it how you have in the photo but remove the map and add post button

Reply:
I like the redirection idea! It's a lot cleaner I think, so I'm changing my vote haha

Reply:
but the link is definitely being removed from the nav bar with option D right?